### ISSUE TITLE
Add a photo via gallery not possible

### ISSUE THREAD

**Issue by Wurftaube** [stefan-niedermann/nextcloud-deck#1609]
### This issue respects the following points:

- [X] I have read the [FAQ](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/FAQ.md)
- [X] I have searched for existing issues
- [X] The issue affects the Deck Android app, neither the [Deck server app](https://github.com/nextcloud/deck/issues/) nor the [Nextcloud Android app](https://github.com/nextcloud/android/issues).

### Describe the bug

1. Go to card
2. Click on "attachements"
3. Click on "gallery"
4. See a white screen with Google Play Service Logo in the upper left corner an below the text:
Ein Fehler ist aufgetreten. Versuch es später noch einmal. (An error has occurred. Try again later.)
Bottom right corner "Cancel" button.
5. Happens every time

### Expected behavior

Opening the gallery app on my phone or a hint, if another Nextcloud app is required.

### Deck Android version

1.24.1

### Deck server version

1.11.4

### Nextcloud Android version

3.28.0

### Nextcloud version

27.1.5 Enterprise 

### Device

Samsung Galaxy Note 9

### Android Version

10

### App Store

- [ ] Google Play Store
- [ ] Google Play Store (Beta channel)
- [X] F-Droid
- [ ] Huawei App Gallery

**Comment by stefan-niedermann** [comment]
Do you have installed Google Play Services? We migrated to the standardized Photo picker provided by Google: https://developer.android.com/training/data-storage/shared/photopicker#device-availability

I am afraid that we currently do not have any plans to support Android 10 and lower without Play Services being installed. Use the file picker as workaround.

> See a white screen with Google Play Service Logo in the upper left corner an below the text:
Ein Fehler ist aufgetreten. Versuch es später noch einmal. (An error has occurred. Try again later.)
Bottom right corner "Cancel" button.

A screenshot might help us to catch this case fall back to the file picker.

**Comment by Wurftaube** [comment]
Ja, ich habe Google Play Dienste installiert, aber keinen Google-Account angemeldet. Den Screenshot habe ich angehängt. 
Ich kann mit dem File Picker leben, jetzt, da ich weiß, dass es an meiner Android-Version liegt. Ich habe sehr häufig Crashes in der Deck-App für Android. Weitere Bug-Reports folgen von Fall zu Fall.

Yes, I have installed Google Play services, but have not registered a Google account. I have attached the screenshot. 
I can live with the file picker now that I know it's due to my Android version. I have very frequent crashes in the Deck app for Android. More bug reports will follow on a case by case basis.


-------- Ursprüngliche Nachricht --------
Von: Niedermann IT-Dienstleistungen ***@***.***>
Gesendet: 26. März 2024 14:14:27 UTC
An: stefan-niedermann/nextcloud-deck ***@***.***>
CC: Wurftaube ***@***.***>, Author ***@***.***>
Betreff: Re: [stefan-niedermann/nextcloud-deck] Add a photo via gallery not possible (Issue #1609)

Do you have installed Google Play Services? We migrated to the standardized Photo picker provided by Google: https://developer.android.com/training/data-storage/shared/photopicker#device-availability

I am afraid that we currently do not have any plans to support Android 10 and lower without Play Services being installed. Use the file picker as workaround.

> See a white screen with Google Play Service Logo in the upper left corner an below the text:
Ein Fehler ist aufgetreten. Versuch es später noch einmal. (An error has occurred. Try again later.)
Bottom right corner "Cancel" button.

A screenshot might help us to catch this case fall back to the file picker.
